corporate/lib/stripe.py

```diff
@@ -1170,6 +1189,50 @@ def set_required_plan_tier(self, required_plan_tier: int) -> str:
             plan_tier_name = CustomerPlan.name_from_tier(new_plan_tier)
         return f"Required plan tier for {self.billing_entity_display_name} set to {plan_tier_name}."
 
+    def create_fixed_price_plan(self, fixed_price: int) -> str:
+        customer = self.get_customer()
```

Comment:
Should we add `AuditLogs` here?

Reply:
They are arguably created in `process_initial_upgrade` so I am not sure but could be helpful for debugging if we had them here.

Reply:
Added auditlogs. I think it's better to add here as we do for other /support actions.

Reply:
Added Auditlog here.

Reply:
Added auditlogs.

> They are arguably created in process_initial_upgrade so I am not sure but could be helpful for debugging if we had them here.

When a customer on billable plan is scheduled t o upgrade to a fixed price plan, we don't go though the `process_initial_upgrade` codepath. So added auditlog here.